test/TestProjects/TypeSchemaMapping/TypeSchemaMapping.json

```diff
@@ -292,7 +292,7 @@
         }
       },
       "x-csharp-usage": "model,input,output",
-      "x-csharp-formats": "xml, json"
+      "x-csharp-formats": "xml, json, json"
```

Comment:
Is this for testing that duplicates don't show up?

Reply:
Yes